front-end/src/app/shared/models/user.model.ts

```diff
@@ -12,6 +13,30 @@ export type UserLoginData = {
   token: string | null;
 };
 
+export class CommitteeUser {
+  email: string | null = null;
+  username: string | null = null;
+  first_name: string | null = null;
+  last_name: string | null = null;
+  role: string | null = null;
+  is_active: boolean | null = null;
+
+  // prettier-ignore
+  static fromJSON(json: any): CommitteeUser { // eslint-disable-line @typescript-eslint/no-explicit-any
+    return plainToClass(CommitteeUser, json);
+  }
+
+  public getRoleLabel(): string | null {
+    const role: string | null = this.role;
```

Comment:
looks like that opens up a tiny can of worms.  for one it would be `LabelUtils.get()` instead of `LabelList.get`.  Two, you'd need to change the arguments of `get` to `get(labelArrays: LabelList, key: string | null)`.  Third now the function returns `''` instead of `null`.  I still stand by my suggestion in a previous pr on returning null on a miss ([reason](https://stackoverflow.com/a/1626620/18957717)), but in this scenario `''` seems rational enough since we're just going to throw the result into the html.

Seems like a lot of work but it reduces code duplication/divergence and you'd be improving the LabelUtil.get which i'm positive we will be using many more times in the app

Reply:
looks like that opens up a tiny can of worms. for one it would be LabelUtils.get() instead of LabelList.get. Two, you'd need to change the arguments of get to get(labelArrays: LabelList, key: string | null). Third now the function returns '' instead of null. I still stand by my suggestion in a previous pr on returning null on a miss ([reason](https://stackoverflow.com/a/1626620/18957717)), but in this scenario '' seems rational enough since we're just going to throw the result into the html.

Seems like a lot of work but it reduces code duplication/divergence and you'd be improving the LabelUtil.get which i'm positive we will be using many more times in the app

Reply:
It does reduce code duplication, so it's worth doing.  Plus, the only thing relying on the method returning null is the unit tests, so those can be changed.

The User model still has the `.getRoleLabel()` method, but it's just a wrapper that returns null if the object's `.role` attribute is set to null and otherwise returns the result from `LabelUtils.get()`

Reply:
And after further consideration, the null-handling is now handled by changes to `LabelUtils.get()` in line with Todd's suggestions